Sources/xcbeautify/root.swift

```diff
@@ -14,68 +14,56 @@ private func configuration(command: Command) {
               value: false,
               description: "Only print tasks that have warnings or errors.",
               inheritable: true)
-        ])
+    ])
     
     command.add(flags: [
         .init(longName: "quieter",
               value: false,
               description: "Only print tasks that have errors.",
               inheritable: true)
-        ])
-
+    ])
+    
+    command.add(flags: [
+        .init(longName: "isCI",
+              value: false,
+              description: "Print test result too under quiet/quiter flag")
+    ])
+    
     command.add(flags: [
         .init(shortName: "v",
               longName: "version",
               value: false,
               description: "Prints the version",
               inheritable: true)
-        ])
-
+    ])
+    
     command.inheritablePreRun = { flags, args in
         if let versionFlag = flags.getBool(name: "version"), versionFlag == true {
-            print(version)
+            print("Version: \(version)")
             return false
         }
-
+        
         return true
     }
-
+    
     command.example = """
-      $ xcodebuild test -project MyApp.xcodeproj -scheme MyApp -destination 'platform=iOS Simulator,OS=12.0,name=iPhone X' | xcbeautify
+    $ xcodebuild test -project MyApp.xcodeproj -scheme MyApp -destination 'platform=iOS Simulator,OS=12.0,name=iPhone X' | xcbeautify
     """
 }
 
 private func execute(flags: Flags, args: [String]) {
+    
     let parser = Parser()
     let quiet = flags.getBool(name: "quiet") == true
     let quieter = flags.getBool(name: "quieter") == true
-    var lastFormatted: String? = nil
-
+    let isCI = flags.getBool(name: "isCI") == true
```

Comment:
Command-line flags should be all lowercase, unless they're short flags.

```suggestion
    let isCI = flags.getBool(name: "is-ci") == true
```